docs/data_engineers/accessing_data/elt_pipeline/elt_destinations/postgres.md

```diff
@@ -45,87 +45,88 @@ In this step, you will deploy a local `Postgres` database using Docker.
    GRANT ALL ON SCHEMA public to airbyte_user;
    ```
 
+   ```sql
+   \quit
+   ```
+
    A database named `kyvetest` has been created and the user `airbyte_user` with password `pass` now
    has read and write access.
 
-4. Install the [pgAdmin 4](https://www.pgadmin.org/download/) management tool.
+4. Install the [pgAdmin 4](https://www.pgadmin.org/download/) Postgres management tool, for example with Homebrew:
 
-5. Click on "Add New Server" and create a connection with the `kyvetest` database.
-   As show in the figure bellow you should add the following parameters:
+   ```sh
+   brew install --cask pgadmin4
+   ```
 
-    - **Maintenance database**: `kyvetest`
-    - **Username**: `airbyte_user`
-    - **Password**: `pass` (in case you did not modify it from step 3.1)
+5. Run pgAdmin and click on "Add New Server". Give it a name, for example `local-psql`, and in the "Connections" tab fill out the fields as follows:
 
-   <br></br>
+   - **Host name/address**: `localhost`
+   - **Maintenance database**: `kyvetest`
+   - **Username**: `airbyte_user`
+   - **Password**: `pass` (in case you did not modify it in step 3)
 
-   <img src="/img/elt/pgAdmin4_connection.jpg"/>
+   <br></br>
+   <img src="/img/elt/pgAdmin4_connection.jpg" alt="pgAdmin Add New Server" />
 
 ## Configure Airbyte
 
-Now you are ready to go on the Airbyte app ([http://localhost:8000/](http://localhost:8000/)) and create a new
-connection.
+Now you're ready to create the connection in Airbyte (<http://localhost:8000/>).
 
 1. Set up the source.
 
-   In this step you should specify the [Kyve pool](https://app.korellia.kyve.network/#/pools) from which you want to
-   retrieve data. You can specify a specific **Bundle-Start-ID** in case you want to narrow the records that will be 
-   retrieved from the pool. You can find the bundles of your pool choice in the KYVE app. For example 
-   [here](https://app.korellia.kyve.network/#/pools/8/bundles) are the bundles of Pool ID `8`, the Evmos chain.
+   In this step you specify the [KYVE pools](https://app.korellia.kyve.network/#/pools) from which you want to retrieve data, identified by **Pool-IDs**.
 
-   For this example we choose:
+   You can also specify **Bundle-Start-IDs** for each pool, in order to limit the records that will be retrieved from that pool. In the KYVE app you can find the bundles for each pool, for example [these](https://app.korellia.kyve.network/#/pools/8/bundles) are the bundles for Pool-ID `8`, the Evmos chain.
```

Comment:
note updates here to plural IDs, since the connector supports multiple Pool ID/Bundle IDs (I'm guessing that updated since the docs were first written?)